include/node.tcc

```diff
@@ -38,6 +38,8 @@ void mpm::Node<Tdim, Tdof, Tnphases>::initialise() noexcept {
   status_ = false;
   solving_status_ = false;
   material_ids_.clear();
+  // mark: need to check
+  discontinuity_enrich_ = false;
```

Comment:
Make initialise_xmpm or whatever. See `initialise_twophase()`

Reply:
will do it in the derived XMPM node class